release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -803,6 +808,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|MaxUnavailableStatefulSet featureset
```

Comment:
There aren't a lot of examples to work from to make me feel one way or the other about this, but did you consider putting this in backticks? In the description above, it is in backticks. I think it _should_ be in backticks since, what I can find out about this, is that it's a configuration parameter. I might be wrong about what it is, though.

Reply:
Seems right, thank you!